pom.xml

```diff
@@ -553,6 +550,12 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- Kotlin -->
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
+      <artifactId>kotlin-stdlib-jdk8</artifactId>
+      <version>${kotlin.version}</version>
+    </dependency>
   </dependencies>
```

Comment:
-1, that's a bridge too far IMO. If we want to create a Kotlin Commons Lang, we can talk about that on the developer's mailing list...